referral/templates/referral/new-patient-contact.html

```diff
@@ -28,7 +31,7 @@ <h1>New referral followup</h1>
 <div class="container">
 	<div class="row">
 		<div class='col-md-offset-2 col-md-8' style='padding-bottom: 2em'>
-		  <p>A referral is considered <strong><span class='text-success'>successful</span> only once a patient has both made <i>and gone to</i> and appointment</strong>. Please <strong><span class='text-info'>continue following up</span></strong> with patients until they have completed an appointment. Only once you have exhausted followup should you save as an <strong><span class="text-danger">unsuccessful referral.</span></strong></p>
+		  <p>{% blocktrans %}A referral is considered{% endblocktrans %} <strong><span class='text-success'>{% blocktrans %}successful{% endblocktrans %}</span> {% blocktrans %}only once a patient has both made{% endblocktrans %} <i>{% blocktrans %}and gone to{% endblocktrans %}</i> {% blocktrans %}and appointment{% endblocktrans %}</strong>. {% blocktrans %}Please{% endblocktrans %} <strong><span class='text-info'>{% blocktrans %}continue following up{% endblocktrans %}</span></strong> {% blocktrans %}with patients until they have completed an appointment{% endblocktrans %}. {% blocktrans %}Only once you have exhausted followup should you save as an{% endblocktrans %} <strong><span class="text-danger">{% blocktrans %}unsuccessful referral{% endblocktrans %}.</span></strong></p>
```

Comment:
This is quite verbose--there are a lot of `blocktrans`s in here... is there really no way to have it just be one block?

Reply:
I have to try that one out. But as far as I remember the html-tags inside the `blocktrans` tags are slipping into the translation, this can be confusing and could lead to errors while translating (e.g. not closing the tag in the right manner).